crates/biome_css_parser/src/syntax/selector/mod.rs

```diff
@@ -212,11 +214,19 @@ pub(crate) fn parse_selector(p: &mut CssParser) -> ParsedSyntax {
         return Absent;
     }
 
-    // In CSS, we have compound selectors and complex selectors.
-    // Compound selectors are simple, unseparated chains of selectors,
-    // while complex selectors are compound selectors separated by combinators.
-    // After parsing the compound selector, it then checks if this compound selector is a part of a complex selector.
-    parse_compound_selector(p).and_then(|selector| parse_complex_selector(p, selector))
+    if p.options().is_grit_metavariable_enabled() {
+        p.re_lex(CssReLexContext::GritMetavariable);
```

Comment:
I'm not sure we need the re-lexing here, because we aren't doing any backwards operation. Instead, you should use parsing with a different context

Reply:
We could try to move this logic inside the lexer. 
By passing a new option to the lexer and, without any additional context, if the next token is `μ` (Επιτέλους, τα ελληνικά μου ήταν χρήσιμα!) and the option is enabled, we can process it as a meta variable.

Reply:
Make sense. Addressed in https://github.com/biomejs/biome/pull/3340/commits/0ca8bd941a52e778a9b44e5140f7d852cfebaf8b